lib-multisrc/mangaworld/src/eu/kanade/tachiyomi/multisrc/mangaworld/CookieRedirectInterceptor.kt

```diff
@@ -13,6 +13,10 @@ class CookieRedirectInterceptor(private val client: OkHttpClient) : Interceptor
 
     override fun intercept(chain: Interceptor.Chain): Response {
         val request = chain.request()
+        val host = request.url.host
+        if (host.startsWith("cdn", ignoreCase = true)) {
+            return chain.proceed(request)
+        }
```

Comment:
this may not work if url doesn't have cdn in it, you could  instead add a special fragment to page image urls and check for it here instead